operator/README.md

```diff
@@ -136,6 +136,13 @@ It is important to proceed with caution when deleting a Karmada CR due to the po
 kubectl delete karmada karmada-demo -n test
 ```
 
+If you want to delete a Karmada CR without cascading deletion of the associated Karmada instance,
+you can execute the following command before performing the deletion operation.
```

Comment:
```suggestion
you can run the following command before performing the deletion operation.
```
Shorter is better in addition to style consistency:

![image](https://github.com/karmada-io/karmada/assets/79828097/dccd6f65-33fc-47ee-a4cc-9ad8a6a19b7c)

Reply:
thanks,done